docs/admin/config.rst

```diff
@@ -1219,18 +1247,25 @@ SUGGESTION_CLEANUP_DAYS
 
 .. versionadded:: 3.2.1
 
-Automatically delete suggestions after given number of days. Defaults to
-``None`` what means no deletion at all.
+Automatically deletes suggestions after given number of days.
+Defaults to ``None``, meaning no deletions.
 
 .. setting:: URL_PREFIX
 
 URL_PREFIX
 ----------
 
 This settings allows you to run Weblate under some path (otherwise it relies on
-being executed from webserver root). To use this setting, you also need to
-configure your server to strip this prefix. For example with WSGI, this can be
-achieved by setting ``WSGIScriptAlias``. The prefix should start with a ``/``.
+being run from the webserver root).
+
+.. note::
+
+To use this setting, you also need to configure your server to strip this prefix.
+For example with WSGI, this can be achieved by setting ``WSGIScriptAlias``.
```

Comment:
```suggestion
    To use this setting, you also need to configure your server to strip this prefix.
    For example with WSGI, this can be achieved by setting ``WSGIScriptAlias``.
```